lldb/source/Plugins/SymbolFile/DWARF/DWARFUnit.h

```diff
@@ -38,7 +39,7 @@ enum DWARFProducer {
   eProducerOther
 };
 
-class DWARFUnit : public UserID {
+class DWARFUnit : public UserID, public DWARFExpression::Delegate {
```

Comment:
```suggestion
class DWARFUnit : public DWARFExpression::Delegate, public UserID {
```

(because Delegate is a "true" base class (with virtual methods and whatnot) while UserID is a mixin)

Reply:
Done.